pkg/apis/build/v1alpha1/build_pod.go

```diff
@@ -3,7 +3,7 @@ package v1alpha1
 import (
 	"encoding/json"
 	"fmt"
-
```

Comment:
Can we fix this import order?

Reply:
ya is need ti fix goland, it keeps messing them up